lms/djangoapps/certificates/queue.py

```diff
@@ -274,7 +274,10 @@ def add_cert(self, student, course_id, course=None, forced_grade=None, template_
             template_pdf = "certificate-template-{id.org}-{id.course}-verified.pdf".format(id=course_id)
         elif mode_is_verified and not user_is_verified:
             template_pdf = "certificate-template-{id.org}-{id.course}.pdf".format(id=course_id)
```

Comment:
I don't want to disrupt the default behavior. I think there is still an expectation amongst course staff that honor certs to be generated when the student did not have their ID verified.  To me this is the least impactful change.
Maybe when we ready to fully move forward, this distinction (honor vs audit) would be no longer valid.